consensus/reactor_test.go

```diff
@@ -235,7 +235,7 @@ func TestReactorCreatesBlockWhenEmptyBlocksFalse(t *testing.T) {
 
 	// send a tx
 	if err := assertMempool(css[3].txNotifier).CheckTx([]byte{1, 2, 3}, nil); err != nil {
-		//t.Fatal(err)
+		t.Error(err)
```

Comment:
why Error and not Fatal?

Reply:
In testing, I found that t.fatal, since it exits, crashes some tests so i went for error